phase1/AbdulWahab/PythonProject(LoadCSVtoSQL)/main.py

```
@@ -0,0 +1,33 @@
+from urllib.parse import quote
+from sqlalchemy import MetaData,create_engine 
+from functionality import load_data,create_table,add_chunks_to_sql
+import os
+import logging
+from pathlib import Path
+
+
+def main():
+    user = '********'
+    password = quote("********")
+    host = '********' 
+    main_folder = 'files'
+    main_folder_path = Path(main_folder)
+    if not main_folder_path.exists():
+        logging.error(f"The directory '{main_folder}' does not exist.")
```

throw error here instead of continuing